src/video/video_reader.cc

```diff
@@ -281,6 +286,20 @@ bool VideoReader::Seek(int64_t pos) {
     return ret >= 0;
 }
 
+bool VideoReader::SeekStart() {
```

Comment:
can we merge this into a special case of Seek(0) in `Seek`?

Reply:
Good suggestion! I have substituted the SeekStart() by Seek(0) and the decoding speed remains similar.